lib/Basics/FileUtils.cpp

```diff
@@ -305,18 +305,14 @@ void spit(std::string const& filename, StringBuffer const& content, bool sync) {
   spit(filename, content.data(), content.length(), sync);
 }
 
-bool remove(std::string const& fileName, int* errorNumber) {
-  if (errorNumber != nullptr) {
-    *errorNumber = 0;
-  }
-
-  int result = std::remove(fileName.c_str());
+bool remove(std::string const& fileName) {
```

Comment:
I would really be in favor of changing this function's return value from `bool` to `ErrorCode`.

Reply:
Good point, done.